sources/ProfilePreferencesViewController.m

```diff
@@ -155,7 +155,12 @@ - (void)awakeFromNib {
         static const CGFloat kMaxHeight = 438;
         if (view.frame.size.height > kMaxHeight) {
             // If the view is too tall, wrap it in a scroll view.
-            NSRect theFrame = NSMakeRect(0, 0, view.frame.size.width, kMaxHeight);
+            
+            CGFloat widthWithScrollBar = view.frame.size.width +
+            [NSScroller scrollerWidthForControlSize:NSRegularControlSize scrollerStyle:NSScrollerStyleLegacy];
```

Comment:
Break this into two lines, aligning the colons (when readability isn't hurt, lines are kept under 100 characters)